src/reader/mqtt_reader.py

```diff
@@ -71,3 +71,15 @@ def read(self, topic, **rdg):
             return None
 
         return self._current_payloads.get(topic)
+    
+    def read_wt_subscribe(self, topic, **rdg):
```

Comment:
I don't entirely understand what's happening here, but I suspect it's stretching the envelope of the (already pretty funky) MQTT reader module. I suggest we talk about it before we pursue this path further.

The MQTT reader module was designed to work in the context of the datalogger taking recurring readings from other brokers on the network (the context of the `get_readings.py` module). Since this in itself is a bit counter to the event-driven pub-sub paradigm, it does some gymnastics in this regard (documented in the comments). Some of these may be appropriate for this use case, but others may not be. If it doesn't directly do what's necessary, I would caution against building on top of these gymnastics.

I have a few ideas of how we can implement this in a fairly robust way. One of them is to simply always store the latest readings in an in-memory KV cache (https://github.com/ammpio/ammp-edge/blob/main/src/kvstore/kv.py#L82), and retrieve them from there - rather than having to mess with MQTT stuff here.